elf_reader.go

```diff
@@ -1120,6 +1135,18 @@ func (ec *elfCode) loadVirtualDataSections() error {
 		return nil
```

Comment:
Get rid of loadVirtualDataSections and inline the calls to loadKconfigSection, etc. into the caller. The function doesn't add much except a level of indirection.